typescript/cli/src/send/message.ts

```diff
@@ -96,6 +96,15 @@ async function executeDelivery({
   );
   const mailbox = core.getContracts(origin).mailbox;
 
+  let hook: Address;
+  try {
+    hook = mergedContractAddrs[origin].customHook;
+    logBlue(`Using custom hook ${hook} for ${origin} -> ${destination}`);
+  } catch (e) {
+    hook = await mailbox.defaultHook();
+    logBlue(`Using default hook ${hook} for ${origin} -> ${destination}`);
+  }
```

Comment:
I recommend avoiding try/catch for control flow like this.
If you're trying to guard against errors from accessing `customHook`, use optional chaining instead:
`mergedContractAddrs[origin]?.customHook`

Reply:
how does the optional chaining help here `mergedContractAddrs[origin]?.customHook` will still be undefined because mergedContractAddrs[origin] is ok but there's no customHook

Reply:
When else would `  hook = mergedContractAddrs[origin].customHook;` throw and error? Why do you have a try catch around it?

Reply:
fixed :)